MidasCivil_Adapter/Convert/ToBHoM/Properties/ToMetaData.cs

```diff
@@ -0,0 +1,132 @@
+                    reviews.Add(review1);
+                }
+                else if (dataItem.Contains("CHECK2="))
+                {
+                    review2.Reviewer = (dataItem.Split('=')[1]);
+                    reviews.Add(review2);
+                }
+                else if (dataItem.Contains("CHECK3="))
+                {
+                    review3.Reviewer = (dataItem.Split('=')[1]);
+                    reviews.Add(review3);
+                }
+                else if (dataItem.Contains("APPROVE="))
+                {
+                    review4.Reviewer = (dataItem.Split('=')[1]);
+                    reviews.Add(review4);
+                    review4.Approved = true;
+                }
+                if (dataItem.Contains("CDATE1=")) { review1.ReviewDate = MCEngine.Convert.Date(dataItem.Split('=')[1]); }
+                else if (dataItem.Contains("CDATE2=")) { review2.ReviewDate = MCEngine.Convert.Date(dataItem.Split('=')[1]); }
+                else if (dataItem.Contains("CDATE3=")) { review3.ReviewDate = MCEngine.Convert.Date(dataItem.Split('=')[1]); }
+                else if (dataItem.Contains("ADATE=")) { review4.ReviewDate = MCEngine.Convert.Date(dataItem.Split('=')[1]); }
+                else if (dataItem.Contains("COMMENT="))
+                {
+                    if (!dataItem.Contains("COMMENT=This Model Was Created Using BHoM Version:"))
+                    {
+                        if (!string.IsNullOrEmpty(review4.Reviewer))
+                        {
+                            comments.Add(dataItem.Split('=')[1]);
+                            review4.Comments = comments;
+                        }
+                        else if (!string.IsNullOrEmpty(review3.Reviewer))
+                        {
+                            review3.Comments.Add(dataItem.Split('=')[1]);
+                            review3.Comments = comments;
+                        }
+                        else if (!string.IsNullOrEmpty(review2.Reviewer))
+                        {
+                            review2.Comments.Add(dataItem.Split('=')[1]);
+                            review2.Comments = comments;
+                        }
+                        else if (!string.IsNullOrEmpty(review1.Reviewer))
+                        {
+                            review1.Comments.Add(dataItem.Split('=')[1]);
+                            review1.Comments = comments;
+                        }
+                        else
+                        {
+                            review1.Reviewer = "Anonymous";
+                            review1.Comments = comments;
+                        }
+                    }
+                }
+            }
+            reviews.Reverse();
+            metadata.Reviews = reviews;
+
+            List<Metadata> returnMetadata = new List<Metadata>(1);
+            returnMetadata.Add(metadata);
+
+            return returnMetadata;
```

Comment:
```suggestion
            return new List<Metadata>() { metadata };
```

Reply:
Lets not be wasteful and have an open limit on our lists, specifying its limit keeps it tidy.
Secondly, pet hate of mine is when the returned variable is created on the return line, makes it really hard to debug.